tensorflow/tools/compatibility/tf_upgrade_v2.py

```diff
@@ -440,6 +440,12 @@ def __init__(self):
             "tensor": "data",
             "family": None,
         },
+        "tf.nn.weighted_cross_entropy_with_logits": {
+            "targets": "labels",
+        },
+        "tf.nn.sparse_softmax_cross_entropy_with_logits": {
```

Comment:
I don't think we need _sentinel here since no one is allowed to actually pass anything as the _sentinel